Gruntfile.js

```diff
@@ -965,7 +963,6 @@ module.exports = function(grunt) {
     'install-dependencies',
     'static-analysis',
     'build',
-    'build-admin',
```

Comment:
We already `build-admin` in our `build -> build-common` task. It also fails if we attempt to build it at this stage cause browserify doesn't like it when shared libs don't have installed dependencies.